### DISCUSSION TITLE
length of list

### DISCUSSION THREAD

**Discussion by junsukha** [alshedivat/al-folio#2011] [Q&A] [answered]
Hi
how do I remove the limit of the number of posts that shows on a page?
For example, you can see only five posts for each page under a 'blog' section. I want to list all posts on one page. 

Thanks for your help!

**Comment by junsukha** [comment, marked as answer]
Solved. Need to edit `per_page` in `index.html`

**Reply by george-gca** [reply]
Actually this was implemented in #1909

**Reply by junsukha** [reply]
@george-gca  oh `enabled` key is right above `per_page`. lol. How did I miss that. Thank you

**Reply by junsukha** [reply]
@george-gca  wait. is that the one? when I set `enabled` to `false`, I don't see any pages.

**Reply by junsukha** [reply]
@george-gca #1909 part of the code was missing on my end. I added manually and now it works. Thanks!
